hugegraph-dist/scripts/apache-release.sh

```diff
@@ -92,9 +95,10 @@ svn status
 
 ##### 4.3 commit & push files
 if [ "$USERNAME" = "" ]; then
-  svn commit -m "submit files for ${REPO} ${RELEASE_VERSION}"
+  svn commit -m "submit files for ${REPO} ${RELEASE_VERSION}" || exit
```

Comment:
prefer to add "set -ev" in the file header

Reply:
It was added before, but the output content is **too much** and makes a little messy

consider test & add it again

Reply:
ok